meta.yml

```diff
@@ -28,9 +29,31 @@ supported_coq_versions:
   text: Coq 8.8 or later
   opam: '{ >= "8.8" }'
 
+ci_test_dependants: true
+
 target: theories
 namespace: ExtLib
 
+build: |
+  ## Building and installation instructions
+
+  The easiest way to install the latest released version of coq-ext-lib
+  is via [OPAM](https://opam.ocaml.org/doc/Install.html):
+
+  ```shell
+  opam repo add coq-released https://coq.inria.fr/opam/released
+  opam install coq-ext-lib
+  ```
+
+  To instead build and install manually, do:
+
+  ``` shell
+  git clone https://github.com/coq-community/coq-ext-lib
+  cd coq-ext-lib
+  make theories  # or make -j <number-of-cores-on-your-machine> theories
```

Comment:
This `theories` is the only change from template, and should be handled by coq-community/templates#46 rather than copying everything here.

Reply:
Changed.